src/besseli.jl

```diff
@@ -130,6 +130,209 @@ function besseli1x(x::T) where T <: Union{Float32, Float64}
     return z
 end
 
+#####
+#####  Implementation for complex arguments
+#####
+
+function besseli0(z::ComplexF64)
+    isconj = false
+    if real(z) < 0.0
+        z = -z
+    end
+    if imag(z) < 0.0
+        z = conj(z)
+        isconj = true
+    end
+    if abs(z) > 17.5
```

Comment:
you should probably use `abs2` here since it's 3x faster.

Reply:
O ya it's much faster to do that instead.